src/components/sbb-checkbox/sbb-checkbox.tsx

```diff
@@ -218,6 +219,15 @@ export class SbbCheckbox implements ComponentInterface {
       : ', ' + i18nCollapsed[this._currentLanguage];
   }
 
+  private _setParametersFromParentGroup(): void {
+    const parentGroup = this._element.closest('sbb-checkbox-group');
+    if (parentGroup) {
+      this._requiredFromGroup = parentGroup.hasAttribute('required');
+      this._disabledFromGroup = parentGroup.hasAttribute('disabled');
```

Comment:
We need to cover the required=false case. please us isValidAttribute helper
```suggestion
      this._requiredFromGroup = isValidAttribute(parentGroup, 'required');
      this._disabledFromGroup = isValidAttribute(parentGroup, 'disabled');
```